src/routes/register_startup.js

```diff
@@ -17,6 +17,6 @@ module.exports = (req, res) => {
     if (err) {
       throw err;
     }
-    res.redirect('/')
+    res.redirect('/submitMessage')
```

Comment:
how would they return to home ??
you should make the return home button  into a link that returns home on top of that make it so after 10-5 sec it would return home by itself if they dont press return home